models/container.go

```diff
@@ -22,18 +22,21 @@ func init() {
 	Containers = make(map[string]*Container)
 }
 
-func GetLog(ContainerId string) (string, error) {
-	return dockerContainerLogs(ContainerId)
+func GetLog(ContainerId string, tail string) (string, error) {
+	return dockerContainerLogs(ContainerId, tail)
 }
 
-func dockerContainerLogs(ContainerId string) (string, error) {
+func dockerContainerLogs(ContainerId string, tail string) (string, error) {
```

Comment:
camelCase on variable names

Reply:
i don't think other apps need to access it. Do we still want to camelCase this?